README.md

```diff
@@ -1,12 +1,11 @@
-
-<!-- TODO: Place a logo below -->
-<!-- <p align="center">[ A LOGO HERE ]</p> -->
-<h1 align="center">
-   Mavlink Camera Manager
-</h1>
+<a href="https://bluerobotics.com">
```

Comment:
...why this link?

Reply:
ooooops, it was based over a template, will fix

Reply:
Fixed!